report_csv/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Base report csv",
+    "summary": "Base module to create csv report",
+    "author": "Creu Blanca, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/reporting-engine",
+    "category": "Reporting",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "external_dependencies": {"python": ["python-csv"]},
```

Comment:
do we need to depend on python-csv? It seems that only csv is being used in the code which is what the dependency was in v13.